package.json

```diff
@@ -73,7 +73,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "moment": "^2.24.0",
-    "pg": "^7.12.1",
-    "uuid": "^3.3.3"
+    "nanoid": "^2.1.11",
+    "pg": "^7.12.1"

```

Comment:
I usually specify exact version to prevent yarn install to bump version without us knowing. i.e remove the `^` for all packages.